<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBenutzerVerwaltung</class>
 <widget class="QDialog" name="frmBenutzerVerwaltung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbMenue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hauptmenü</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Benutzer Verwaltung&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>14</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBeenden">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>471</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>91</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jahrgang:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchwort:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbJahrgang">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>91</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtSuchwort">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSuchen">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwAnzeigen">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>164</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBearbeiten">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>bearbeiten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoeschen">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>löschen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBenutzerAnlegen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>neu</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAdmin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Admins suchen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImport">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
